.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -2,8 +2,8 @@
 
 1. If this is your first time, please read our contributor guidelines: https://git.k8s.io/community/contributors/guide#your-first-contribution and developer guide https://git.k8s.io/community/contributors/devel/development.md#development-guide
 2. Please label this pull request according to what type of issue you are addressing, especially if this is a release targeted pull request. For reference on required PR/issue labels, read here:
-https://git.k8s.io/community/contributors/devel/release.md#issue-kind-label
-3. Ensure you have added or ran the appropriate tests for your PR: https://git.k8s.io/community/contributors/devel/testing.md
+https://github.com/kubernetes/community/blob/master/contributors/devel/sig-release/release.md#issuepr-kind-label
```

Comment:
please use `https://git.k8s.io/community` instead of `https://github.com/kubernetes/community/blob/master` throughout

Reply:
Hi @liggitt 
Thanks for your review.

So this link should be: "https://git.k8s.io/community/contributors/devel/sig-release/release.md#issuepr-kind-label", right?
Could you please share the reason?
Is this applied for other links in other files as well?

Thank you in advance & wish you a great day ahead :)

Reply:
Yes the convention is to use git.k8s.io so that the `/master/blob` thing isn't exposed to the user. There are plenty of places where we use the uglier URLs but as a minimum we try to at least not introduce them in new documents. This is looking good so far though, just do a quick search and replace and we'll take it from there, thanks!